charts/flink-job/ci/full-values.yaml

```
@@ -224,3 +224,6 @@ ingress:
 
 image-automation:
   enabled: false
+
+local-network-configuration:
+  enabled: true
```

We should keep the camelcase formatting (please ignore `image-automation` :sweat_smile: )
```suggestion
localNetworkConfiguration:
  # -- Create a localNetworkConfiguration for allowing traffic between job- and task-manager in a deny-all environment
  enabled: true

image-automation:
  enabled: false

```